plugins/jira/docs/issue-types/epic.md

```diff
@@ -0,0 +1,498 @@
+/jira:create epic OCPEDGE "Epic summary" --template common-epic
+```
+
+## What is an Epic?
+
+An agile epic is:
+- A **body of work** that can be broken down into specific items (stories/tasks)
+- Based on the needs/requests of customers or end-users
+- An important practice for agile and DevOps teams
+- A tool to **manage work** at a higher level than individual stories
+
+### Epic Characteristics
+
+Epics should:
+- Be **more narrow** than a market problem or feature
+- Be **broader** than a user story
+- **Fit inside a time box** (quarter/release)
+- Stories within the epic should **fit inside a sprint**
+- Include **acceptance criteria** that define when the epic is done
+
+### Epic vs Feature vs Story
+
+| Level | Scope | Duration | Example |
+|-------|-------|----------|---------|
+| Feature | Strategic objective, market problem | Multiple releases | "Advanced cluster observability" |
+| Epic | Specific capability, narrower than feature | One quarter/release | "Multi-cluster metrics aggregation" |
+| Story | Single user-facing functionality | One sprint | "As an SRE, I want to view metrics from multiple clusters in one dashboard" |
+
+**When to use each:**
+- **Feature**: Strategic initiative spanning multiple quarters, requires multiple epics to deliver
+- **Epic**: Specific capability that delivers value but needs multiple stories to implement
+- **Story**: Single piece of user-facing functionality that can be completed in one sprint
+
+## Epic Components
+
+### Summary
+
+The epic summary should be:
+- **Concise**: 5-10 words max
+- **Descriptive**: Clearly states the capability
+- **Action-oriented**: Uses verbs when appropriate
+- **Specific**: Not vague or generic
+
+**Examples:**
+- ✅ "Multi-cluster metrics aggregation for ROSA HCP"
+- ✅ "Enable graceful API server responses during maintenance"
+- ❌ "Improve monitoring" (too vague)
+- ❌ "As an SRE, I want to view metrics..." (that's a story, not an epic)
+
+### Epic Name
+
+**IMPORTANT:** The Epic Name is a separate field from Summary in many Jira configurations.
+
+- Short identifier (3-5 words)
+- Displayed in epic picker and boards
+- Often set to match or abbreviate the summary
+- Missing epic name will cause creation to fail in some projects
+
+**Examples:**
+- Summary: "Multi-cluster metrics aggregation for ROSA HCP"
+  Epic Name: "Multi-cluster Metrics Aggregation"
+
+- Summary: "Enable graceful API server responses during maintenance"
+  Epic Name: "API Graceful Responses"
+
+### Objective
+
+The epic objective should:
+- State what capability will be delivered
+- Explain why it matters
+- Identify who benefits
+- Be concise and direct
+
+**Writing Guidelines:**
+- **Be concise**: Remove unnecessary adjectives and adverbs
+- **Be direct**: State facts, not opinions or emphasis
+- **Be specific**: Use concrete nouns and verbs
+- **Avoid filler words**: "really", "very", "easily", "simply", "just", "actually", "comprehensive", "powerful"
+
+**Good example (concise, direct):**
```

Comment:
I'm not a big fan of good and bad. Let's update this and add a policy for using strong and weak instead. This removes the implications of an moral assessments.

Reply:
I thought about this more and I couldn't come up with anything that doesn't have some kind of moral assessement. Good and bad are as good as anything else I thought of, and simpler.